cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -163,6 +163,9 @@ public String toString() {
     "'<namespace-name>.<artifact-name>.<artifact-version>' or " +
     "'<namespace-name>.<app-name>.<app-version>'.\n" +
     "\n" +
+    "Note:  Metadata for versioned entities such as applications, programs, schedules and program runs is not " +
+    "versioned. If a metadata is added to one specific version it will be reflected to all versions." +
```

Comment:
"If a metadata" --> "If metadata".
Same in the other file.

Reply:
Not needed with above change.